### DISCUSSION TITLE
how to get dates

### DISCUSSION THREAD

**Discussion by andresrogers** [alvarobartt/investiny#19] [Q&A] [answered]
when downloading historical data, how can you get the date for each value?

for example:
start_date = '04/27/2000'
end_date = date.today().strftime("%m/%d/%Y")
historical_data(investing_id=8830, from_date=start_date, to_date=end_date)

returns only the gold prices, but can't establish a date for each value

**Comment by alvarobartt** [comment, marked as answer]
Hi, @andresrogers can you please try the latest `investiny` release?

Install it as `pip install investiny==0.5.0` and then run the following curated version of your code:

```python
from investiny import historical_data
from datetime import datetime

data = historical_data(investing_id=8830, from_date="04/27/2000", to_date=datetime.today().strftime("%m/%d/%Y"))
print(data) # Python dictionary that contains the keys: date, open, high, low, close, and volume
```

Hope this helps! I've tested it on my own and it works fine :hugs:

P.S. note that you're formatting the current date as `%d/%m/%Y` while `investiny`'s date should be formatted as `%m/%d/%Y`.

**Comment by andresrogers** [comment]
It works with the latest update, thanks!

Do you know if there is a limit of 5000 datapoints? Although I used to_date= "today" it gives me 5000 rows with the latest being 12/02/2019

**Reply by alvarobartt** [reply]
Mmm, I was not aware of that, but nice catch, can you please open a separate issue at https://github.com/alvarobartt/investiny/issues? Thanks 👍🏻

**Reply by varshasahasrabuddhe** [reply]
How can i use investpy.economic_calendar ?
